src/components/Filter.js

```diff
@@ -1,13 +1,21 @@
 import { FilterBar, Button } from './StyledComponents';
 
-function Filter() {
+function Filter(props) {
+    let filterInput;
+
     return (
         <div>
             <FilterBar 
                 type="text" 
-                placeholder="Filter by name" 
+                placeholder="Filter by name"
+                onChange={evt => filterInput = evt.target.value} 
```

Comment:
I don't like to use on change here. There is no need to do this. I think you can create a search button and use it.